types/formdata.d.ts

```diff
@@ -0,0 +1,97 @@
+// based on https://unpkg.com/browse/formdata-node@4.0.1/@type/FormData.d.ts (MIT)
```

Comment:
```suggestion
// based on https://github.com/octet-stream/form-data (MIT)
```

Reply:
I changed it, but on second thought I think it was better before